source/common/http/http1/codec_impl.h

```diff
@@ -162,7 +163,6 @@ class RequestEncoderImpl : public StreamEncoderImpl, public RequestEncoder {
 private:
   bool upgrade_request_{};
   bool head_request_{};
-  bool connect_request_{};
```

Comment:
why the rename?  we're only tracking CONNECT requests, not connect responses right?

Reply:
yep